Source/DafnyLanguageServer/Language/IVerificationProgressReporter.cs

```diff
@@ -20,5 +20,6 @@ public interface IVerificationProgressReporter {
     void ReportEndVerifyImplementation(Implementation implToken, Boogie.VerificationResult verificationResult);
     void ReportImplementationsBeforeVerification(Implementation[] implementations);
     void ReportAssertionBatchResult(AssertionBatchResult batchResult);
+    void ReportAssertionBatchResult(Implementation implementation, VCResult batchResult);
```

Comment:
Why are there two `ReportAssertionBatchResult` ? Seems like you only need the new one.